Terminal.Gui/Core/EscSeqUtils/EscSeqReq.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Terminal.Gui {
+	/// <summary>
+	/// Represents a state of escape sequence request.
+	/// </summary>
+	/// <remarks>
+	/// This is needed because there are some escape sequence requests responses that are equal
+	/// with some normal escape sequences and thus, will be only considered the responses to the
+	/// requests that were registered with this object.
+	/// </remarks>
+	public class EscSeqReqStat {
+		/// <summary>
+		/// Gets the terminating.
```

Comment:
What is a "terminating"? ("Terminating" is not a noun). The terminating "what"?

Reply:
Is the last string on a escape ansi sequence. This terminology is normal used in theses cases. See the unit tests and you'll understand better what I mean.